scripts/nb-tester/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = [ "hatchling",]
+build-backend = "hatchling.build"
+
+[project]
+name = "qiskit-docs-notebook-tester"
+version = "0.0.1"
+description = "Tool for the Qiskit docs team to test their notebooks"
+requires-python = ">=3.8"
```

Comment:
We use Python 3.9 syntax, which imo is fine for our tutorials. See https://github.com/Qiskit/documentation/pull/1210

But you'll need to require 3.11+ if you want to use `tomllib` since it wasn't in the std lib. You could instead use `tomli` or `tomlkit`. (`toml` is not maintained.) `tomli` looks good to me: https://pypi.org/project/tomli/. Technically this would only impact people running `nb-tester`, but if we're not running our notebooks with 3.9 in CI then it's likely we'll unintentionally use code that breaks with 3.9 in our notebooks.